$OpenBSD: patch-fixincludes_fixincl_x,v 1.1 2011/12/09 22:45:18 pascal Exp $
Generated from inclhack.def.  Do not edit this file directly; instead,
patch inclhack.def, regenerate this file using "genfixes" and then generate
a patch from it.
--- fixincludes/fixincl.x.orig	Fri Dec  9 15:32:02 2011
+++ fixincludes/fixincl.x	Fri Dec  9 15:32:08 2011
@@ -2,11 +2,11 @@
  * 
  * DO NOT EDIT THIS FILE   (fixincl.x)
  * 
- * It has been AutoGen-ed  Monday November  8, 2010 at 08:33:47 PM MET
+ * It has been AutoGen-ed  Friday December  9, 2011 at 03:18:29 PM CET
  * From the definitions    inclhack.def
  * and the template file   fixincl
  */
-/* DO NOT SVN-MERGE THIS FILE, EITHER Mon Nov  8 20:33:47 MET 2010
+/* DO NOT SVN-MERGE THIS FILE, EITHER Fri Dec  9 15:18:29 CET 2011
  *
  * You must regenerate it.  Use the ./genfixes script.
  *
@@ -6256,12 +6256,14 @@ tSCC* apzSolaris_Complex_CxxMachs[] = {
  *  Fix Command Arguments for Solaris_Complex_Cxx
  */
 static const char* apzSolaris_Complex_CxxPatch[] = { sed_cmd_z,
-    "-e", "/#if[ \t]*!defined(__cplusplus)/c#ifdef\t__cplusplus\\\n\
+    "-e", "/#if[ \t]*!defined(__cplusplus)/c\\\n\
+#ifdef\t__cplusplus\\\n\
 extern \"C\" {\\\n\
-#endif",
-    "-e", "/#endif[ \t]*\\/\\* !defined(__cplusplus) \\*\\//c#ifdef\t__cplusplus\\\n\
+#endif\\\n",
+    "-e", "/#endif[ \t]*\\/\\* !defined(__cplusplus) \\*\\//c\\\n\
+#ifdef\t__cplusplus\\\n\
 }\\\n\
-#endif",
+#endif\\\n",
     (char*)NULL };
 
 /* * * * * * * * * * * * * * * * * * * * * * * * * *
